test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -363,14 +362,18 @@ public void CreateDescriptor_BuildsDescriptorsWithInheritedProperties()
         {
             // Arrange
             var errorSink = new ErrorSink();
-            var intProperty = typeof(InheritedSingleAttributeTagHelper).GetProperty(
-                nameof(InheritedSingleAttributeTagHelper.IntAttribute));
+
+            // Also confirm isStringProperty is calculated correctly.
```

Comment:
Maybe I'm missing something, aren't these comments only valuable in the case that two `TagHelperDescriptor`s target the same attribute with different types?